tests/utils.go

```diff
@@ -494,7 +494,7 @@ func WaitForAllPodsReady(timeout time.Duration, listOptions metav1.ListOptions)
 }
 
 func SynchronizedAfterTestSuiteCleanup() {
-	RestoreKubeVirtResource()
+	RestoreKubeVirtResource(originalKV)
```

Comment:
I find using this unscoped variable weird. I don't know where it comes from, nor who initializes it.